### ISSUE TITLE
Problem on notification from command line [Mydd3]

### ISSUE THREAD

**Issue by francescobianco** [Myddleware/myddleware#811]
Because notifications weren't working, we did a test from CLI

If i run from CLI the following command:

```shell
php bin/console myddleware:notification ALL
```

I get back an error and **not email was sent**

```text
15:50:44 ERROR     [app] Error : [Semantical Error] line 0, col 93 near 'log INNER JOIN': Error: Class App\Entity\Job has no association named log /var/www/html/vendor/doctrine/orm/lib/Doctrine/ORM/Query/QueryException.php Line : ( 43 )
Error : [Semantical Error] line 0, col 93 near 'log INNER JOIN': Error: Class App\Entity\Job has no association named log /var/www/html/vendor/doctrine/orm/lib/Doctrine/ORM/Query/QueryException.php Line : ( 43 )
```

**Comment by CamillePMyddleware** [comment]
Hello @francescobianco,
Sorry for the delay, we are going to make some tests on our side.
We will keep you posted as soon as possible.

thanks and have a nice day

**Comment by simonetravaglini** [comment]
@CamillePMyddleware hi, news about this topic? You were able to reproduce issue?

**Comment by CamillePMyddleware** [comment]
Hi @simonetravaglini,
Sorry for the delay, for the moment we are trying to reproduce the problem, we will come back to you as soon possible.

thanks

**Comment by simonetravaglini** [comment]
@CamillePMyddleware @Myddleware any news? Becouse we are not able to send notification from M3 at all... :(

**Comment by Myddleware** [comment]
Hi @simonetravaglini ,
Just let you know taht notifications are fixed in the next main release (3.1.8). We try to finish all the test tomorrow then we will launch it.
We will keep you posted.

**Comment by AlexMyddleware** [comment]
@simonetravaglini i marked this issue as resolved as in the current myddleware release i just tested the notifications test and the command, it works. Feel free to reopen this issue if you have further problems.
